README.md

```diff
@@ -166,28 +166,38 @@ deploy:
 1. Install arduino-cli from here: https://arduino.github.io/arduino-cli/installation/
 2. Download ci-arduino
    * `git clone https://github.com/adafruit/ci-arduino`
-3. Put these lines at the end of your `.bashrc` or `.bash_profile` if you're on OSX. Make sure to fill in the path to where you installed ci-arduino and replacing USER with your username.
+3. Put these lines at the end of your `.bashrc` or `.bash_profile` if you're on OSX. Make sure to fill in the path to where you installed ci-arduino and replacing USER with your username. Make sure to that the libraries directory exists by installing a library first:
+   * `arduino-cli lib install "Adafruit NeoPixel"` 
+   * Linux
    ```bash
    alias test-platforms='python3 ~/path/to/ci-arduino/build_platform.py'
-   export HOME=/home/USER/
+   export HOME=/home/USER
    ```
+   * Mac/OSX
+   ```bash
+   alias test-platforms='python3 ~/path/to/ci-arduino/build_platform.py'
+   export HOME=/Users/USER
+   export ARDUINO_LIB_DIR=/Documents/Arduino/libraries
+   ```
+   * Then run `source .bashrc` (Linux) or `source .bash_profile` (Mac/OSX)
```

Comment:
Should be `source ~/.bashrc`

Docker output:
```root@7a4055dfae8e:/# source .bashrc
bash: .bashrc: No such file or directory
root@7a4055dfae8e:/# source ~/.bashrc```

Reply:
@brentru If it's mac you have to use .bash_profile. Although I am not sure if that comes pre-created on OSX

Reply:
I think what brent was saying is that the path should be given within the home directory: `~/.bashrc` instead of just `.bashrc`.

Reply:
Oh you're completely right lol